qa/workunits/cephtool/test.sh

```diff
@@ -486,7 +486,7 @@ function test_auth()
   ceph auth print-key client.xx
   ceph auth print_key client.xx
   ceph auth caps client.xx osd "allow rw"
-  expect_false "ceph auth get client.xx | grep caps | grep mon"
+  expect_false sh <<< "ceph auth get client.xx | grep caps | grep mon"
```

Comment:
@tchaikov 
With pipe fial I thing you could write this as:
`expect_false sh (backtick)ceph auth get client.xx | grep caps | grep mon"(backtick)`

Sorry about the `(backtick)`, but I don't know how to get a ` code format.